sql/2025/fonts/design/fonts_designer.sql

```diff
@@ -0,0 +1,54 @@
+-- Section: Design
+-- Question: Which designers are popular?
+-- Normalization: Pages
+
+-- INCLUDE https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/sql/{year}/fonts/common.sql
+
+WITH
+designers AS (
+  SELECT
+    client,
+    NULLIF(TRIM(STRING(JSON_QUERY(payload, '$._font_details.names[9]'))), '') AS designer,
+    COUNT(DISTINCT page) AS count,
+    ROW_NUMBER() OVER (PARTITION BY client ORDER BY COUNT(DISTINCT page) DESC) AS rank
```

Comment:
`rank` is a confusing column name since we also use this for [CrUX Rank](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide#rank)

Maybe change to just `row`? Or `row_number`? Note if changing you'll need to change the `QUALIFY` clause too.

Reply:
Here we assume the writer knows the context. I think the probability of confusion is small. As with `count`, `total`, and `proportion`, I have been trying to stay parsimonious with names. So `rank` is the only name used for ranking results in all queries.